### ISSUE TITLE
Jet Smart Filters visual filter icons FA5 and Custom Icons

### ISSUE THREAD

**Issue by stasonua0** [Crocoblock/suggestions#988]
Hello, very often you need to add exactly the icons from the FA5 set or from the custom icon set created in Elementor. Please consider the possibility of implementing icons in visual filters
https://prnt.sc/rcipz5

Здравствуйте, очень часто нужно добавить именно иконки из набора FA5 или из кастомного набора иконок созданного в Elementor. Рассмотрите пожалуйста возможность внедрения иконок в визуальные фильтры
https://prnt.sc/rcipz5

**Comment by ALPSIT** [comment]
Yes please! it would be great to be able to upload custom SVG icon and select Normal, Active and Hover colours for Visual Radio and Checkbox filters.

**Comment by ALPSIT** [comment]
![image](https://user-images.githubusercontent.com/85008592/163914185-8a1826c4-3cdc-4015-a72c-61bbbe9851ba.png)
This screenshot above is a bit misleading. There is no Icon Picker. Only Color Picker or Image

**Comment by wpraiz** [comment]
Please, add this feature guys.

**Comment by natequreshi** [comment]
I really want this feature to have icons/svg with each taxonomy to show in search filter, yet there is nothing but I am sure a small code might work.

**Comment by atikrahman146** [comment]
+1

**Comment by mennybe** [comment]
Was that publish? Any turnaround to create this? I tried HTML img tag but it broke the title visuality.

**Comment by Kokouliss** [comment]
Please add the icon feature. it would be very useful for the icon to change color on hover/checked state

**Comment by IslamSultan** [comment]
+

**Comment by IslamSultan** [comment]
+++
